src/components/DownloadImageDialog/index.tsx

```diff
@@ -66,6 +66,26 @@ const getUniqueOsTypes = (
 	return uniq(osVersions[deviceTypeSlug].map((x) => x.osType));
 };
 
+const isUrlAccessible = async (url: string, authToken: string | undefined) => {
```

Comment:
Perhaps this should go to DownloadImageDialog utils or to ui-shared's global utils